test/e2e/pods.go

```diff
@@ -982,46 +981,6 @@ var _ = Describe("Pods", func() {
 		}
 	})
 
-	It("should not back-off restarting a container on LivenessProbe failure [Serial]", func() {
```

Comment:
I discussed with @dchen1107 , and we didn't quite understand why kubelet `should not back-off restarting a container on LivenessProbe failure`, and in fact the test is a little out-of-date [here](https://github.com/kubernetes/kubernetes/pull/22241#issue-137440699). So I just removed it.